docs/preview/features/logging.md

```diff
@@ -114,11 +130,33 @@ public class Startup
         });
 
         ...
+
         app.UseMvc();
     }
 }
 ```
 
+**Skipping certain routes**
+
+The request tracking middleware can be disabled for certain endpoints (operation and/or controller level). This can come in handy if you have certain endpoints with rather large request bodies or want to boost performace.
```

Comment:
```suggestion
You can opt-out for request tracking on one or more endpoints (operation and/or controller level).

This can come in handy if you have certain endpoints with rather large request bodies or want to boost performance or are constantly probed to monitor the application.
```